packages/devtools_app/lib/src/shared/release_notes/release_notes.dart

```diff
@@ -38,7 +38,8 @@ class ReleaseNotesViewer extends StatefulWidget {
 
 class _ReleaseNotesViewerState extends State<ReleaseNotesViewer>
     with AutoDisposeMixin, SingleTickerProviderStateMixin {
-  static const viewerWidth = 600.0;
+  static const maxViewerWidth = 600.0;
+  static const animationStartValue = 1000.0;
```

Comment:
honestly it is arbitrary. There is no specific reason for it being 1000, other than it seeming like a nice number.
I will experiment to see if I can just set the animation to go from 0.0 to 1.0 though. as this would simplify some math